### ISSUE TITLE
ON HOLD - Ant Design Modals Are Not Read Aloud By Screen Reader by Narrator (does by NVDA)

### ISSUE THREAD

**Issue by ianwilk20** [CDH-Studio/UpSkill#1277]
**Describe the bug**

Ant Design Modals do not read out the contained information through a screen reader like Windows Narrator

**To Reproduce**
1. Go to the I-Talent Homepage
2. Click on the "Report a bug" button
3. Hear the screen reader not read out the majority of the "Location" buttons, the body of the modal, the greyed-out text of the modal

**Expected behavior**
The modal should read out the information of the entire dialog/modal

- [ ] Report a Bug Modal
- [ ] Settings Permanently Delete Account Modal
- [ ] Login Privacy Modal
- [ ] Geds Update Modal
- [ ] General Search Matches Modal (When searching for something in basic search and clicking on file-search-outlined icon)
- [ ] Welcome Page on First time Account Setup Modal
- [ ] Admin Bug Table Modal
- [ ] Admin Category Page Modal
- [ ] Admin Competency Page Modal
- [ ] Admin Diploma Page Modal
- [ ] Admin School Page Modal
- [ ] Admin Skill Page Modal

**Comment by Rizvida1** [comment]
The modals and app work well with NVDA but with Windows Narrator, the screen reader gets stuck on an endless loop and doesn't read the modal so until we know which screen reader to optimize for, this should be on hold.

**Comment by dynamic11** [comment]
Please see if #1653 or other related pull request by me resolved this or will help resolve this
